vulnfeeds/cpp/main.go

```diff
@@ -179,6 +180,9 @@ func ReposForCPE(CVE string, cache VendorProductToRepoMap, vp VendorProduct, ref
 			// Failed to parse as a valid repo.
 			continue
 		}
+		if !git.ValidRepo(repo) {
+      continue
```

Comment:
Please fix indentation.

Reply:
GitHub's merge conflict web UI for the lose... Fixed in 
52b0285